forte/processors/data_augment/algorithms/dictionary_replacement_augmenter.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+from typing import Dict, List
+from forte.processors.data_augment.algorithms.base_augmenter import BaseDataAugmenter
```

Comment:
We follow PEP8 that group the imports into 3 groups, and add new line between the groups. Check here: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
This was part of Yu's commit, but I'll keep that in mind.